app/models/edm/web_resource.rb

```diff
@@ -119,6 +121,10 @@ def rdf_about
       media&.url
     end
 
+    def remove_versions
+      media.versions.each { |key, _version| media.send(key).remove! }
```

Comment:
Is this necessary? Are they otherwise left behind?

Is the full-size/original media object destroyed automatically?

Reply:
indeed, on deletion the original media is automatically deleted, however versions aren't. Hence this method.